docs/web-apps/serverless/architectures/web-app-content.md

```diff
@@ -13,7 +13,7 @@ The term serverless has two distinct but related meanings:
 - **Backend as a service (BaaS)**. Back-end cloud services, such as databases and storage, provide APIs that enable client applications to connect directly to these services.
 - **Functions as a service (FaaS)**. In this model, a "function" is a piece of code that is deployed to the cloud and runs inside a hosting environment that completely abstracts the servers that run the code.
 
-Both definitions have in common the idea that developers and DevOps personnel don't need to deploy, configure, or manage servers. This reference architecture focuses on FaaS using Azure Functions, although serving web content from Azure Blob Storage could be an example of BaaS. Some important characteristics of FaaS are:
+Both definitions have in common the idea that developers and DevOps personnel don't need to manage servers. Also, both definitions imply that those services can scale to zero and that users pay only for what they use. This reference architecture focuses on FaaS using Azure Functions, although serving web content from Azure Blob Storage could be an example of BaaS. Some important characteristics of FaaS are:
```

Comment:
This is only true if it's priced that way.  It's not a feature of the platform, it's a function of pricing of that platform component.  Both of these points are added already in the following three bullet points, so I'm not seeing anything novel being addressed here.  I'm going to close this PR, but feel free to reopen if you want to bring a novel thought into this section by reworking this a bit more.